agent/utils/ticker_test.go

```diff
@@ -0,0 +1,66 @@
+package utils
+
+import (
+	"context"
+	"math/rand"
+	"testing"
+	"time"
+)
+
+const (
+	TheBestNumber = 28
+)
+
+func TestTickerHappyCase(t *testing.T) {
+	ctx, _ := context.WithTimeout(context.Background(), 1000 * time.Millisecond)
+	mTicker := JitteredTicker(ctx, 10*time.Millisecond, 100*time.Millisecond)
+
+	times := 0
+	for {
+		_, ok := <-mTicker
+		times++
+		if !ok {
+			break
+		}
+	}
+
+	if 10 > times || times > 100  {
```

Comment:
nit: can you change it to `times<10 || times > 100`?

Reply:
Sure! I rewrote this a couple times because i couldn't decide what looked best.. :smile: